src/app/window-handler.ts

```diff
@@ -315,11 +334,18 @@ export class WindowHandler {
             }
         });
 
+        this.mainWindow.once('closed', () => {
+            this.destroyAllWindow();
```

Comment:
please refactor the method name to be `destroyAllWindows` in place of `destroyAllWindow`